packages/@lwc/errors/src/__tests__/errors.spec.ts

```diff
@@ -75,4 +77,22 @@ describe('error validation', () => {
 
         traverseErrorInfo(CompilerErrors, checkUniqueness, 'compiler');
     });
+
+    it('Next error code is updated', () => {
```

Comment:
should we add info about where to add the new error code (`../compiler/error-info/index.ts`)?

Reply:
done